source/main/modules/fsal/fsal-file.js

```diff
@@ -222,11 +222,14 @@ function parseFileContents (file, content) {
   while ((match = tagRE.exec(mdWithoutCode)) != null) {
     let tag = match[1]
     tag = tag.replace(/#/g, '') // Prevent headings levels 2-6 from showing up in the tag list
-    if (tag.length > 0) file.tags.push(match[1].toLowerCase())
+    if (tag.length > 0) file.tags.push(match[1].toLocaleLowerCase())
   }
 
   // Merge possible keywords from the frontmatter
   if (file.frontmatter && file.frontmatter.hasOwnProperty('keywords')) {
+    // Lower-case (as with regex-extracted tags) and replace spaces with dashes
```

Comment:
This will confuse users as it will mess with people's expectations, so I would really like you to remove this `map`. We need a better way to do that (and I may have one, see my yet-to-be-written comment at the end)

Reply:
Are both lower-casing and hyphenation a problem, or just the hyphenation? I'd retain the map to do the lower-casing.